.github/workflows/on_build_do_publish_image.yml

```diff
@@ -63,19 +63,14 @@ jobs:
               targetUrl: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
   publish-image:
-      needs:
-         - get-info
-      if: needs.get-info.outputs.publish-image == 'true'
-      uses: infinispan/infinispan-images/.github/workflows/publish_image.yml@main
-      secrets:
-         token: ${{ secrets.GITHUB_TOKEN }}
-         quayUser: ${{ secrets.QUAY_USERNAME_TEST }}
-         quayPass: ${{ secrets.QUAY_TOKEN_TEST }}
-      with:
-         repository: ${{ github.repository }}
-         runId: ${{ github.event.workflow_run.id }}
-         serverArtifact: infinispan-dist
-         tags:  ${{ needs.get-info.outputs.image-name }}:${{ needs.get-info.outputs.image-tag }}
+    needs:
```

Comment:
If we want this to work with the old images, I think we need to introduce an additional job controlled by an `if` statement. Something like `publish-image-legacy`.